plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -257,4 +258,16 @@ public PinotConfig setMaxRowsPerSplitForSegmentQueries(int maxRowsPerSplitForSeg
         this.maxRowsPerSplitForSegmentQueries = maxRowsPerSplitForSegmentQueries;
         return this;
     }
+
+    public boolean getForbidDropTable()
+    {
+        return forbidDropTable;
+    }
+
+    @Config("pinot.forbid-drop-table")
```

Comment:
We should implement the standard security modules for Pinot instead of doing it this way. See this for an example: https://github.com/trinodb/trino/pull/588/commits/f660baff5020ef2262534418ea35d48c5e2a6b7d

Note that we should use `allow-all` rather than `none` (I need to update the above) and we should probably default to `read-only` so that all of the write features are disabled by default.